vignettes/MDA.Rmd

```diff
@@ -15,22 +15,39 @@ knitr::opts_chunk$set(
 ```
 
 ```{r setup}
-suppressPackageStartupMessages(library(ggplot2))
-library(malariasimulation)
-library(malariaEquilibrium)
-library(reshape2)
+# Load the requisite packages:
+library(malariasimulation); library(malariaEquilibrium)
+
+# Define a colour palette for plotting:
+plot_cols <- c("#E69F00", "#56B4E9", "#009E73", "#CC79A7","#F0E442", "#0072B2", "#D55E00")
+
 ```
 
-# Parameterisation
+## Introduction
+
+Malaria chemoprevention encompasses a suite of strategies for reducing malaria morbidity, mortality, and transmission through the treatment of at-risk populations with antimalarial drugs. Three such strategies are mass drug administration (MDA), seasonal malaria chemoprevention (SMC), and perennial malaria chemoprevention (PMC). MDA attempts to simultaneously treat an entire population, irrespective of their infection risk or status. SMC targets children aged 3-59 months with a course of an antimalarials through the peak malaria transmission season. PMC is designed for settings with perennially high transmission and seeks to administer a full antimalarial course at defined intervals irrespective of age or disease status.
+
+`malariasimulation` enables users to simulate the effects of chemoprevention strategies on malaria transmission, and provides the flexibility to design and simulate a range of MDA/SMC/PMC campaigns for a suite of antimalarial drugs. Specifically, users are able to load or specify parameters describing both the characteristics of the antimalarial(s) (e.g. drug efficacy, prophylactic profile) and their distribution (e.g. coverage, distribution timing, and the section of the population to treat). This vignette provides simple, illustrative demonstrations of how MDA and SMC campaigns can be set-up and simulated using `malariasimulation`. The functionality to simulate PMC campaigns is also built into `malariasimulation`, but we do not demonstrate that here.
```

Comment:
Hi Tom, I think it is fine to leave out an example of PMC but I'm glad you mention it in this intro text. Just add into the last sentence a link to the function `set_pmc` and say that documentation about setting up PMC can be found there

Reply:
I've amended the final sentence as follows to direct readers towards the documentation for the set_pmc() function:

"The functionality to simulate PMC campaigns is also built into `malariasimulation` via the `set_pmc() function`, but we do not demonstrate the use of PMC in this vignette (for further information on their use, see https://mrc-ide.github.io/malariasimulation/reference/set_pmc.html)."